yourgov/components/FormMobileFoodVendorPermit/task/FormTaskStep1.tsx

```diff
@@ -107,7 +107,7 @@ export function FormTaskStep1() {
 }
 
 function AdditionalDetailsForm() {
-	const { formState, setFormState, isSavingBeforeExiting } = useGlobalForm();
+	const { formState, step1SetState, isSavingBeforeExiting } = useGlobalForm();
```

Comment:
I think `setStep1State` would be easier to read and understand

Reply:
By itself, yes but this aligns with our concentric naming. e.g. this file is `FormTaskStep1`. It also allows more obvious group naming if we choose to extend the hook suite:

```ts
{
  step1SetState,
  step1GetState,
  step1Validate,
  step2SetState,
  step2GetState,
  step2Validate,
};
```